tweepy/cache.py

```diff
@@ -129,7 +130,7 @@ class FileCache(Cache):
     """File-based cache"""
 
     # locks used to make cache thread-safe
-    cache_locks = {}
+    cache_locks: Dict[str, threading.Lock] = {}
```

Comment:
`dict` can be used here with `from __future__ import annotations`.

Reply:
ed58b61